src/main/java/com/revealprecision/revealserver/fhir/properties/FhirServerProperties.java

```diff
@@ -0,0 +1,16 @@
+package com.revealprecision.revealserver.fhir.properties;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConfigurationProperties(prefix = "fhir.server")
+@Setter
+@Getter
+public class FhirServerProperties {
+
+  private String baseURL = "http://localhost:8080";
```

Comment:
will this setup work in production?

Reply:
These are able to be overridden with a config on application.properties.